ledger/catchpointtracker.go

```diff
@@ -610,14 +613,14 @@ func (ct *catchpointTracker) commitRound(ctx context.Context, tx trackerdb.Trans
 }
 
 func (ct *catchpointTracker) postCommit(ctx context.Context, dcc *deferredCommitContext) {
+	ct.catchpointsMu.Lock()
```

Comment:
It seems like we now protect `ct.balancesTrie` with the mutex, where before we didn't.  If that's the intent, let's correct this comment.

```
	// catchpointsMu protects `roundDigest`, `reenableCatchpointsRound` and
	// `lastCatchpointLabel`.
	catchpointsMu deadlock.RWMutex
```

Reply:
done